SU-SWS/decanter
# Adds Tugboat and fixes npm dependencies.

# READY FOR REVIEW

# Summary
- Adds tugboat
- Fixes npm dependencies for webpack-cli vs webpack-command

# Review By (Date)
- Sooner the better

# Urgency
- Low but helpful.

# Steps to Test

1. Check out this branch
2. Run `nvm use` 
3. Run `npm install`
4. Run `npm run build`
5. Run `npm run styleguide`
6. Run `npm run start`

# Affected Projects or Products
- Just Decanter and the various PRs.

# Associated Issues and/or People
- None

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
